server/index.js

```diff
@@ -1,9 +1,29 @@
 const express = require("express");
-
+const bodyParser = require("body-parser");
+const db = require("../database/index.js");
+// const cors = require("cors");
 let app = express();
 
+// app.use(cors());
+app.use(express.static(__dirname + "/../client/dist/"));
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.get("/review_id/", (req, res) => {
+  const restaurantId = req.params.id;
```

Comment:
When you get reviews, why is the path called 'review_id' when then id you are specifying is for the restaurant?

Reply:
@ bill i will make the changes so that my get request to make it sound better